config/jobs/kubernetes/sig-testing/make-test.yaml

```diff
@@ -22,7 +22,7 @@ periodics:
             - |
               result=0
               # Run the tests as usual
-              GO111MODULE=off go install k8s.io/kubernetes/vendor/gotest.tools/gotestsum || result=$?
+              ( cd hack/tools && GO111MODULE=on go install gotest.tools/gotestsum ) || result=$?
```

Comment:
updated to include this one as well
this should probably get DRYed out but it's a prototype soo :man_shrugging: